### PR TITLE
Add ssl bool

### PR DESCRIPTION
**Pull request by rutgercap**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def version():
     install_requires=[
         "requests >= 2.28",
         "urllib3 >= 1.26",
-        "aiohttp >= 3.8.3",
+        "aiohttp >= 3.9.0b0",
```
Comment (benbrandt) [inline setup.py:47]: Why do we need a beta version?

File: aleph_alpha_client/aleph_alpha_client.py [hunk 2]
```diff
@@ -615,6 +624,7 @@ def __init__(
             start_timeout=0.25,
             statuses=set(RETRY_STATUS_CODES),
         )
+        connector = aiohttp.TCPConnector(ssl=verify_ssl)
```
[the remark this answers is not in the document]
Comment (rutgercap) [reply aleph_alpha_client/aleph_alpha_client.py:627]: I might have misread this! Will check tomorrow morning.

File: setup.py (hunk 1 above)
Comment (rutgercap) [reply setup.py:47]: Aiohttp didn't run on latest version of python in CI. According to a stackoverflow post python 3.11 and/or 3.12 support is still in beta. Mostly wanted to try out if this was true. It seemed to fix the problem, but it broke CI on python 3.7 in exchange for that

Comment (rutgercap) [reply setup.py:47]: should have closed the PR afterwards

Comment (benbrandt) [reply setup.py:47]: Fun lol. Ok no worries

**Pull request closed without merging**